os/include/tinyara/net/net.h

```diff
@@ -280,6 +273,138 @@ void net_setup(void);
 
 void net_initialize(void);
 
+/* Only UDS uses net_lock */
```

Comment:
how about to create new file for net_lock instead of adding it net.h header

Reply:
Okay